### ISSUE TITLE
Hangs taking power of decorated interval

### ISSUE THREAD

**Issue by dpsanders** [dpsanders/ValidatedNumerics.jl#194]
```
julia> x = @decorated(-1, 2)
DecoratedInterval(Interval(-1.0, 2.0), com)

julia> x^Interval(3..4)
```
hangs

**Comment by lbenet** [comment]
You are right; I'll try to take a look on this...

**Comment by lbenet** [comment]
We had only implemented `x^y` with `x` and `y` being `DecoratedInterval`. See #195

**Comment by dpsanders** [comment]
Fixed by #195
